src/ethereum_test_specs/base.py

```diff
@@ -119,5 +139,51 @@ def get_next_transition_tool_output_path(self) -> str:
             str(current_value),
         )
 
+    def is_slow_test(self) -> bool:
+        """Check if the test is slow."""
+        if self._request is not None and hasattr(self._request, "node"):
+            return self._request.node.get_closest_marker("slow") is not None
+        return False
+
+    def is_negative_test(self) -> bool | None:
+        """
+        Check if the test is an exception test (invalid block, invalid transaction).
+
+        `None` is returned if it's not possible to determine if the test is negative or not.
+        This is the case when the test is not run in pytest.
+        """
+        if self._request is not None and hasattr(self._request, "node"):
+            return self._request.node.get_closest_marker("exception_test") is not None
+        return None
+
+    def node_id(self) -> str:
+        """Return the node ID of the test."""
+        if self._request is not None and hasattr(self._request, "node"):
+            return self._request.node.nodeid
+        return ""
+
+    def check_negative_test(
```

Comment:
maybe change to match the new naming?
```suggestion
    def check_exception_test(
```

Reply:
Good catch, thanks!